From 603e9ed58a23a27a92dc457cf31ac4b054d962b4 Mon Sep 17 00:00:00 2001
From: Kexy Biscuit <kexybiscuit@aosc.io>
Date: Sat, 25 Jan 2025 02:57:18 +0800
Subject: [PATCH 203/318] AOSCOS: MIPS: Loongson-3: depends on BROKEN to
 disable

Disable CPU_HWMON (Loongson 3A/B CPU hardware monitoring), as it is
known to cause thermal shutdowns from erratic temperature readings - we
observed this on Excelsior E23L board (3B4000) boards. We should
re-enable this once we find the root cause to this behaviour.

Suggested-by: Mingcong Bai <jeffbai@aosc.io>
Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/platform/mips/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/platform/mips/Kconfig b/drivers/platform/mips/Kconfig
index fb4ac4b08e89..73431d65e143 100644
--- a/drivers/platform/mips/Kconfig
+++ b/drivers/platform/mips/Kconfig
@@ -20,6 +20,7 @@ if MIPS_PLATFORM_DEVICES
 config CPU_HWMON
 	bool "Loongson-3 CPU HWMon Driver"
 	depends on MACH_LOONGSON64
+	depends on BROKEN
 	select HWMON
 	default y
 	help
-- 
2.48.1

